weasyprint/css/validation.py

```diff
@@ -1267,6 +1269,23 @@ def bookmark_level(token):
         return 'none'
 
 
+@validator(prefixed=True)  # CSS3 GCPM, experimental
+def string_set(tokens):
+    """Validation for ``string-set``."""
+    if len(tokens) != 2:
+        raise InvalidValues("Only one value is supported. Received %s" %len(tokens)-1)
+    function = parse_function(tokens[1])
+    if function is None:
+        keyword = 'text'
```

Comment:
When the second token is not a function, it could be anything else. This should check what it is and only accept the expected syntax.